.circleci/config.yml

```diff
@@ -0,0 +1,170 @@
+          ssh -i ~/shopsys www-data@heimdall.shopsys.com rm -rf /workspace/shopsys/$CIRCLE_BRANCH/source
+          ssh -i ~/shopsys www-data@heimdall.shopsys.com sudo chown -R www-data:www-data /workspace/shopsys/$CIRCLE_BRANCH/db
+          ssh -i ~/shopsys www-data@heimdall.shopsys.com rm -rf /workspace/shopsys/$CIRCLE_BRANCH/db/*
+          scp -i ~/shopsys source.tar.gz www-data@heimdall.shopsys.com:/workspace/shopsys/$CIRCLE_BRANCH/source.tar.gz
+          ssh -i ~/shopsys www-data@heimdall.shopsys.com tar -zxvf /workspace/shopsys/$CIRCLE_BRANCH/source.tar.gz -C /workspace/shopsys/$CIRCLE_BRANCH/
+          ssh -i ~/shopsys www-data@heimdall.shopsys.com rm /workspace/shopsys/$CIRCLE_BRANCH/source.tar.gz
+          ssh -i ~/shopsys www-data@heimdall.shopsys.com mkdir -p /workspace/shopsys/$CIRCLE_BRANCH/db
+          ssh -i ~/shopsys www-data@heimdall.shopsys.com chown -R www-data:www-data /workspace/shopsys/$CIRCLE_BRANCH
+
+    - run:
+        name: Deploy application to Kubernetes cluster
+        command: |
+          kubectl delete namespace $CIRCLE_BRANCH || true
+          kubectl create namespace $CIRCLE_BRANCH || true
+          kubectl config set-context $(kubectl config current-context) --namespace=$CIRCLE_BRANCH
+          kubectl apply --recursive -f .
+        working_directory: project-base/k8s
+
+    - run:
+        name: Wait for rollouts
+        command: |
+          kubectl rollout status deployment/adminer --watch
+          kubectl rollout status deployment/postgres --watch
+          kubectl rollout status deployment/redis --watch
+          kubectl rollout status deployment/redis-admin --watch
+          kubectl rollout status deployment/selenium-server --watch
+          kubectl rollout status deployment/smtp-server --watch
+          kubectl rollout status deployment/webserver --watch
+    - store_artifacts:
+        path: project-base/k8s
+    - store_artifacts:
+        path: project-base/app/config
+
+    - run:
+        name: Install composer dependencies
+        command: |
+          PHP_FPM_POD=$(kubectl get pods -l app=webserver,commit=$CIRCLE_SHA1 -o=jsonpath='{.items[0].metadata.name}')
+          kubectl exec $PHP_FPM_POD -c php -- composer config -g github-oauth.github.com $COMPOSER_GITHUB_OATUH_TOKEN
+          kubectl exec $PHP_FPM_POD -c php -- composer global require hirak/prestissimo
+          kubectl exec $PHP_FPM_POD -c php -- composer install --optimize-autoloader --no-interaction --no-progress
+
+    - run:
+        name: Create databases, build application and run tests
+        command: |
+          PHP_FPM_POD=$(kubectl get pods -l app=webserver,commit=$CIRCLE_SHA1 -o=jsonpath='{.items[0].metadata.name}')
+          kubectl exec $PHP_FPM_POD -c php ./phing db-create test-db-create build-demo-ci
+
+workflows:
+  version: 2
+  deployment:
+    jobs:
+    - deploy-to-k8s:
+        filters:
+          tags:
+            ignore: /.*/
```

Comment:
Please add a note saying that this is done to prevent tags from building and why.

Reply:
Also, please make sure that all the config files contain exactly one EOL and the end for more readable diffs (damn, we should have created a sniff for this already).